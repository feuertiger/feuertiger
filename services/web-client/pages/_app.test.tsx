import { test, describe, expect } from '@jest/globals';
import { shallow } from 'enzyme';
import App, { AppProps } from './_app';

const appPropsMock: AppProps = {
    auth: {
        signInWithEmailAndPassword: () => {},
        signOut: () => {}
    },
    error: undefined,
    apollo: {},
    isLoading: false,
    isSignedIn: true,
    pageProps: {}
};

describe('Test nextjs app', () => {
    test('should create _app', () => {
        const component = shallow(
            <App {...appPropsMock}>
                <div>test</div>
            </App>
        );
        expect(component).toMatchSnapshot();
    });
});
