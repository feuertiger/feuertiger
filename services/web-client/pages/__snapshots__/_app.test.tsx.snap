// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test nextjs app should create _app 1`] = `
<AuthWrapper
  apollo={Object {}}
  auth={
    Object {
      "signInWithEmailAndPassword": [Function],
      "signOut": [Function],
    }
  }
  isLoading={false}
  isSignedIn={true}
  pageProps={Object {}}
>
  <div>
    test
  </div>
</AuthWrapper>
`;
